daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/scenario/ScenarioLedger.scala

```diff
@@ -284,7 +289,7 @@ object ScenarioLedger {
   case object OperatorView extends View
 
   /** The view of the ledger at the given party. */
-  final case class ParticipantView(party: Set[Party]) extends View
+  final case class ParticipantView(actAs: Set[Party], readAs: Set[Party]) extends View
```

Comment:
Perhaps worth pointing out that the only reason why we separate the two here is for better error messages. Visibility checks the union.